### PR TITLE
Fix wrong processing of SetDisplayLayout response data.

### LINKED ISSUE #1392: [GENIVI] Wrong processing of SetDisplayLayout response data
# Description
Wrong processing of SetDisplayLayout response data

## Preconditions
SDL is started
Media App is activated

## Steps to reproduce
Send SetDispalyLayout

## Actual result
SetDisplayLayout response from HMI contains parameters: imageFields, imageCapabilities, textFields with value name="timeToDestination". SDL does not send imageCapabilitiesdata to mobile side, name value in imageFields is parsed like next after specified in ImageFieldName enum, name of textFields "timeToDestination" is parsed like 27.

## Expected result 
SDL should send imageCapabilities data to mobile side, name value in imageFields and textFields should be parsed correctly.

## Environment
SDL: HEAD of https://github.com/smartdevicelink/sdl_core/tree/feature/extended_policy_functionality
(commit d5f63b4f66d9ab945d452dfb1c07bf7ee3ca781e)
OS with SDL: Ubuntu 14.04
VM used for OS with SDL: NO
ATF: HEAD of https://github.com/smartdevicelink/sdl_atf/tree/develop
(commit aa5beae6ab1726a9b43ed2fa15f66c20d7f7883f)
Transport: WiFi

### PR DESCRIPTION
**Pull request by IGapchuk**
Fixes #1392

This PR is ready for review.

### Risk
This PR makes major API changes.

### Testing Plan
Covered with atf script.
https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1974

### Summary
MOBILE_API did not match to HMI_API. To MOBILE_API was added several missing values, namely:
enum "TextFieldName": navigationText, notificationText, timeToDestination;
enum "ImageFieldName": imageCapabilities;

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: src/components/interfaces/MOBILE_API.xml [hunk 1]
```diff
@@ -788,6 +795,7 @@
             <description> Optional hone number of intended location / establishment (if applicable) for SendLocation.</description>
         </element>
         
+        <element name="timeToDestination"/>
```
Comment (AKalinich-Luxoft) [inline src/components/interfaces/MOBILE_API.xml]: @IGapchuk please add description for this parameter:
```
<description>Text of time to reach destination to be displayed on screen.</description>
```

File: src/components/interfaces/MOBILE_API.xml [hunk 2]
```diff
@@ -772,6 +772,13 @@
             <description> Optional text to label an app menu button (for certain touchscreen platforms).</description>
         </element>
         
+        <element name="navigationText">
+            <description>Navigation text for UpdateTurnList.</description>
+        </element>
+        <element name="notificationText">
```
Comment (AKalinich-Luxoft) [inline src/components/interfaces/MOBILE_API.xml]: @IGapchuk please insert a blank line to separate these elements

File: src/components/interfaces/MOBILE_API.xml (hunk 1 above)
Comment (IGapchuk) [reply src/components/interfaces/MOBILE_API.xml]: Done. Description was added.

File: src/components/interfaces/MOBILE_API.xml (hunk 2 above)
Comment (IGapchuk) [reply src/components/interfaces/MOBILE_API.xml]: Done. Empty line was added.

**Comment by ValeriiMalkov** [comment]
@IGapchuk Check build on CI

**Comment by LuxoftAKutsan** [comment]
@IGapchuk this parameters was missed? Please add @KhrystynaDubovyk  in the review/

**Comment by robinmk** [comment]
@KhrystynaDubovyk 
Please hold off onto making the change. Let's discuss this item in the weekly meetings on Tue/Thu.

**Comment by iCollin** [comment]
The `TextFieldName` issues will be fixed when [sdl_evolution/#305](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0305-homogenize-textfieldname.md) is implemented.

The `DisplayCapabilities` change may still be needed but it will also require a proposal.

**Pull request closed without merging**